scripts/build/gen_relocate_app.py

```diff
@@ -126,48 +126,48 @@ class OutputSection(NamedTuple):
 
 MPU_RO_REGION_START = """
 
-     _{0}_mpu_ro_region_start = ORIGIN({1});
+     _{memory_type}_mpu_ro_region_start = ORIGIN({memory_type_upper});
 
 """
 
 MPU_RO_REGION_END = """
 
-    _{0}_mpu_ro_region_end = .;
+    _{memory_type}_mpu_ro_region_end = .;
 
 """
 
 # generic section creation format
 LINKER_SECTION_SEQ = """
 
-/* Linker section for memory region {2} for  {3} section  */
+/* Linker section for memory region {memory_type_upper} for {kind_name} section  */
 
-	SECTION_PROLOGUE(.{0}_{1}_reloc,,)
+	SECTION_PROLOGUE(.{memory_type}_{kind}_reloc,,)
         {{
                 . = ALIGN(4);
-                {4}
+                {linker_sections}
                 . = ALIGN(4);
-	}} {5}
-        __{0}_{1}_reloc_end = .;
-        __{0}_{1}_reloc_start = ADDR(.{0}_{1}_reloc);
-        __{0}_{1}_reloc_size = __{0}_{1}_reloc_end - __{0}_{1}_reloc_start;
+	}} {load_address}
+        __{memory_type}_{kind}_reloc_end = .;
+        __{memory_type}_{kind}_reloc_start = ADDR(.{memory_type}_{kind}_reloc);
+        __{memory_type}_{kind}_reloc_size = __{memory_type}_{kind}_reloc_end - __{memory_type}_{kind}_reloc_start;
```

Comment:
when changing from numbers to named placeholders, then please care for line length.

Reply:
Fixed by renaming the `{memory_type}` placeholder to `{mem}`.

Reply:
Replaced `{memory_type}` with `{mem}` so as to be more concise.